bookshelf/resources/views/show.php

```diff
@@ -32,23 +28,24 @@
   </div>
 </form>
 
-{# [START book_details] #}
+<?php // [START book_details] ?>
 <div class="media">
-  {# [START book_image] #}
+  <?php // [START book_image] ?>
+  <?php if ($imgUrl = $book->get('image_url')): ?>
   <div class="media-left">
-    <img class="book-image"
-         src="{{ book.image_url ?: 'http://placekitten.com/g/128/192' }}">
+    <img class="book-image" src="<?= $imgUrl ?>" />
```

Comment:
`<?=` is for output, so unless there's something being printed I use `<?php`. The comments could go either way but I think it makes more sense to keep them as `<?php` since nothing is being output there.